sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -476,6 +446,66 @@ export class EventProcessor {
     this._isRunning = false;
   }
 
+  private async _performLoadBalancing(
+    loadBalancingStrategy: LoadBalancingStrategy,
+    partitionIds: string[],
+    abortSignal: AbortSignalLike
+  ) {
+    if (abortSignal.aborted) return;
```

Comment:
Seems like you could just throw AbortError here. Are there some side effects if you do?

Reply:
I think that's a good point, I'll update to throw AbortError where I'm currently returning instead. I think I had returns in here when this method was inlined with the loop, and just copied that over here.

Reply:
Updated!